src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-detail/cephfs-detail.component.html

```diff
@@ -42,7 +42,7 @@
     </cd-cephfs-clients>
   </tab>
   <tab i18n-heading
-       *ngIf="grafanaPermission.read"
+       *ngIf="grafanaPermission.read && grafanaId"
```

Comment:
If we implement this here, we should implement in all other pages.

Reply:
After testing this change, I would prefer to always show the tab and instead show a different message.
I don't like the idea of tabs showing/hiding depending on the row you click.

Reply:
Done